assets/css/style.css.liquid

```diff
@@ -0,0 +1,1013 @@
+---
+permalink: /css/style.css
+---
+
+@import url("https://fonts.bunny.net/css?family=dm-sans:300,700|space-grotesk:300,400,700");
```

Comment:
It seems we're not importing the right font here, shouldn't we import "Space Grotesk" and "Fira Mono"?

Reply:
Fixed at [2502d23](https://github.com/servo/servo.org/pull/202/commits/2502d237fe04a173d45c9abfe0605513ea4a699f)